packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -270,6 +278,46 @@ const Dropdown = React.forwardRef(
     }: DropdownProps<ItemType>,
     ref: ForwardedRef<HTMLButtonElement>
   ) => {
+    const { refs, floatingStyles } = useFloating(
+      autoAlign
+        ? {
+            // placement: direction,
+
+            // The floating element is positioned relative to its nearest
+            // containing block (usually the viewport). It will in many cases also
+            // “break” the floating element out of a clipping ancestor.
+            // https://floating-ui.com/docs/misc#clipping
+            strategy: 'fixed',
+
+            // Middleware order matters, arrow should be last
+            middleware: [
+              flip({
+                fallbackAxisSideDirection: 'none',
+              }),
```

Comment:
`none` is the default, I think we can remove this

```suggestion
              flip(),
```